src/views/Profile.vue

```diff
@@ -41,6 +40,21 @@
             block
             rounded
           > Copy House Key</v-btn>
+
+          <v-list-item-subtitle class="mt-3 mb-3 font-weight-bold">
+          Or.. Manually copy housekey
```

Comment:
I think the best simple way to handle this is to not add a subtitle at all, and possibly move the housekey display just above the button.

The next step (not for this PR) is to do like Evan suggested, and have a modal "get housekey" popup that shows it and allows you to click to copy. But that's for later.

Reply:
I kept it under the button because the button will work in everything except ios devices. So I do want to keep it as the main option, and the housekey display to be an ios, or failsafe solution.

Reply:
As for the that line, I was having a hard time thinking what to write for that. I think a subtitle is needed, to point out to people that if the button doesn't work, do it manually. I just really couldn't find a good phrase to use....

Reply:
Ok I’m that case, maybe something like “send your house key to new housemates!”

Reply:
I think having the auto copy house key is not a good idea because if it doesn't work on one type of device, we shouldn't implement it at all. Especially if that device is something as popular as an iphone or mac

Reply:
Open to discussion of course.

Reply:
This is a valuable point. Do we know why this doesn't work on all ios devices (it works on mine)?

Reply:
it does work on my phone... odd lol I sware it didn't before